src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -1193,7 +1193,7 @@ objects:
         | **Error on complete** | **Data element to display error next to**<br>         <br>**Tracked entity attribute to display error next to**<br>         <br>**Static text**<br>         <br>**Expression to evaluate and display after static text** | Used whenever you've cross-consistencies in the form that must be strictly adhered to. This action prevents the user from continuing until the error is resolved.<br>         <br>This action differs from the regular **Show error** since the error is not shown until the user tries to actually complete the form.<br>         <br>If you don't select a data element or a tracked entity attribute to display the error next to, make sure you write a comprehensive error message that helps the user to fix the error. |
         | **Hide field** | **Data element to hide**<br>         <br>**Tracked entity attribute to hide**<br>         <br>**Custom message for blanked field** | Used when you want to hide a field from the user.<br>         <br>**Custom message for blanked field** allows you to define a custom message displayed to the user in case the program rule hides and blanks out the field after the user typed in or selected a value.<br>         <br>If a hide field action hides a field that contains a value, the field will always removed. If no message is defined, a standard message will be displayed to alert the user. |
         | **Hide section** | **Program stage section to hide** | Used when you want to hide a section in a program stage from the user.  |
-        | **Prevent adding new events to stage** | **Program stage where new events should be prevented** | Used when you want to prevent new events to be added to a program stage. Existing events will not be hidden.
+        | **Prevent adding new events to stage** | **Program stage where new events should be prevented** | Used when you do not want users to add any more events to a program stage. Existing events will not be hidden.
```

Comment:
Would rename "Program stage where new events should be prevented" as it's the same as the title itself. 

Maybe into: "Restrict users to adding new events to the program stage" 

Otherwise LGTM

Reply:
"Program stage where new events should be prevented" is explaining what you are selecting in the dropdown specifically, so should say "Program stage [action of program rule]". So could be "Program stage where users will not be able to add new events"?

Reply:
Ahh, thank you for explanation! Yes, perfect!

Reply:
I think `Program stage where users will not be able to add new events` is OK, there aren't many other ways to word this in the current pattern we're using.